migrations/000011_delete_partition_function.up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE OR REPLACE FUNCTION drop_ros_partition(tableDate TEXT)
+RETURNS void AS
+$BODY$
+DECLARE
+    allTables TEXT[];
+    tableName TEXT;
+BEGIN
+    SELECT array_agg(partition_table::TEXT) INTO allTables FROM (SELECT relname AS partition_table, matches[1]::date AS min_rangeval, matches[2]::date AS max_rangeval FROM pg_class CROSS JOIN regexp_matches(pg_get_expr(relpartbound, oid), '\((.+?)\).+\((.+?)\)') AS matches WHERE relispartition AND relkind = 'r') nn WHERE nn.min_rangeval < tableDate::date;
```

Comment:
I partially get the query, request to add comments with good explanation. At the same time can we use snake case and better names for variables?

Reply:
Added the comment.